clang/include/clang/Sema/Scope.h

```diff
@@ -236,12 +236,25 @@ class Scope {
 
   void setFlags(Scope *Parent, unsigned F);
 
+  unsigned TargetThreadLimit = 0;
```

Comment:
This will probably need to change (along with all subsequent uses) to support generic expressions and not just constants.

```suggestion
  clang::Expr TargetThreadLimit = nullptr;
```